docs/user/FAQS.md

```diff
@@ -14,6 +14,29 @@ the appropriate repository or start a discussion.
 Thank you for your willingness to contribute. Please see the
 [Getting Involved](../contrib/GettingInvolved) guide.
 
+## How do I update OpenROAD-flow scripts?
+
+Depending on how you installed OpenROAD-flow-scripts, there are different ways of updating. Regardless, here are the common first steps:
+
+```
+cd OpenROAD-flow-scripts
+git checkout master
+git pull
+```
+For WSL/docker based installation, run:
+```
+./build_openroad.sh
+```
+
+For local installation, run:
+```
+./build_openroad.sh --local
+```
+
+For prebuilt binaries, you can run the flow after `git pull` is done.
```

Comment:
```suggestion
For pre-built binaries, you can run the flow after `git pull` is done.
```

Reply:
I believe we settled on using the hyphenated form, please review all docs so we can maintain consistency.